server/config/topic_page/equity/USA.textproto

```diff
@@ -0,0 +1,425 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## This file specifies the topic page config for poverty in USA
+
+metadata {
+  topic_id: 'equity'
+  topic_name: 'Equity'
+  place_dcid: 'country/USA'
+}
+
+blocks {
+  title: "Gini index of economic activity"
+  stat_var_metadata: {
+    stat_var: "GiniIndex_EconomicActivity"
+    name: "Gini index of economic activity"
+  }
+  left_tiles {
+    type: HIGHLIGHT
+    description: "Gini index of economic activity for ${place} (${date})"
+  }
+  left_tiles {
+    type: DESCRIPTION
+    description: "In economics, the Gini index is a measure of income inequality within a nation or social group. It measures the inequality among values of a frequency distribution (e.g. a Gini index of 0 represents perfect equality, while 100 represents perfect inequality)."
+  }
+  right_tiles: {
+    type: LINE
+    title: "Gini index of economic activity for ${place}"
+  }
+  # TODO: Add rankings of other /Countries/
+}
+
+blocks {
+  title: "Income and employment, by sex"
+  description: "Median incomes of individuals, broken down by sex"
+  stat_var_metadata {
+    stat_var: "Median_Income_Person_15OrMoreYears_Female_WithIncome"
+    unit: "$"
+  }
+  stat_var_metadata {
+    stat_var: "Median_Income_Person_15OrMoreYears_Male_WithIncome"
+    unit: "$"
+  }
+  left_tiles {
+    title: "Median income by sex"
+    type: LINE
+  }
+  right_tiles {
+    title: "Unemployment rate by sex"
+    type: LINE
+    stat_var_override {
+      stat_var: "UnemploymentRate_Person_Female"
+      unit: "%"
+    }
+    stat_var_override {
+      stat_var: "UnemploymentRate_Person_Male"
+      unit: "%"
+    }
+  }
+  # right_tiles {
```

Comment:
did you mean to keep this tile commented out?

Reply:
deleted, and added a comment instead.